posthog/tasks/org_usage_report.py

```diff
@@ -154,22 +166,51 @@ def get_org_usage(
         "event_count_lifetime": None,
         "event_count_in_period": None,
         "event_count_in_month": None,
+        "event_count_in_month_vs_previous": None,
+        "session_recording_count_in_month": None,
     }
     usage = default_usage
+    previous_month_start = get_same_date_previous_month(month_start)
+    previous_period_end = get_same_date_previous_month(period_end)
     if data_source == "clickhouse":
         from ee.clickhouse.models.event import get_agg_event_count_for_teams, get_agg_event_count_for_teams_and_period
+        from ee.clickhouse.models.session_recording_event import get_session_recording_count_for_teams_and_period
+
+        event_count_in_current_month = get_agg_event_count_for_teams_and_period(team_ids, month_start, period_end)
+        event_count_in_previous_month = get_agg_event_count_for_teams_and_period(
+            team_ids, previous_month_start, previous_period_end
+        )
 
         usage["event_count_lifetime"] = get_agg_event_count_for_teams(team_ids)
         usage["event_count_in_period"] = get_agg_event_count_for_teams_and_period(team_ids, period_start, period_end)
-        usage["event_count_in_month"] = get_agg_event_count_for_teams_and_period(team_ids, month_start, period_end)
+        usage["event_count_in_month"] = event_count_in_current_month
+        usage["event_count_in_month_vs_previous"] = event_count_in_current_month - event_count_in_previous_month
+        usage["session_recording_count_in_month"] = get_session_recording_count_for_teams_and_period(
+            team_ids, month_start, period_end
+        )
+
     else:
+        event_count_in_current_month = Event.objects.filter(
+            team_id__in=team_ids, timestamp__gte=month_start, timestamp__lte=period_end,
+        ).count()
+        event_count_in_previous_month = Event.objects.filter(
+            team_id__in=team_ids, timestamp__gte=previous_month_start, timestamp__lte=previous_period_end,
+        ).count()
+
         usage["event_count_lifetime"] = Event.objects.filter(team_id__in=team_ids).count()
         usage["event_count_in_period"] = Event.objects.filter(
             team_id__in=team_ids, timestamp__gte=period_start, timestamp__lte=period_end,
         ).count()
-        usage["event_count_in_month"] = Event.objects.filter(
-            team_id__in=team_ids, timestamp__gte=month_start, timestamp__lte=period_end,
-        ).count()
+        usage["event_count_in_month"] = event_count_in_current_month
+        usage["event_count_in_month_vs_previous"] = event_count_in_current_month - event_count_in_previous_month
```

Comment:
Hey @samwinslow we were talking about M/M event growth for this metric - I think that would be best represented as a float...

I was thinking what we'd do is return the value of # events this month / # events last month and return that as a decimal - so anything > 1 would indicate positive growth. I think it would be a lot easier to reason about that than the absolute values since we're charting for growth (and we can use event_count_in_month to get the absolute number still)